services/feature-toggle-service/src/main/java/com/sourcefuse/jarc/services/featuretoggleservice/services/FeatureHandlerService.java

```diff
@@ -0,0 +1,25 @@
+package com.sourcefuse.jarc.services.featuretoggleservice.services;
+
+import com.sourcefuse.jarc.services.featuretoggleservice.annotation.FeatureHandlers;
+import com.sourcefuse.jarc.services.featuretoggleservice.annotation.FeatureToggle;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class FeatureHandlerService {
+
+  @Autowired(required = false)
+  private List<FeatureHandlers> featureHandler;
+
+  public boolean featureHandle(FeatureToggle featureToggle) {
+    for (int i = 0; i < featureHandler.size(); i++) {
```

Comment:
featureHandler is not required right, so it can be null and when its null
featureHandler.size() this may cause the NullPointerException